### PR TITLE
fix a number of uninitialized variable uses

### PR DESCRIPTION
**Pull request by multun**
I did a valgrind run to try and figure out why it OpenBoard crashes from
time to time, and found these two uninitialized variable uses.

### CODE REVIEW

**Comment by multun** [comment]
Just found two more

**Comment by multun** [comment]
?

File: src/domain/UBGraphicsScene.cpp [hunk 1]
```diff
@@ -1052,7 +1053,9 @@ void UBGraphicsScene::eraseLineTo(const QPointF &pEndPoint, const qreal &pWidth)
                 polygonItem->setNominalLine(false);
                 polygonItem->setStroke(intersectedPolygonItem->stroke());
                 polygonItem->setStrokesGroup(intersectedPolygonItem->strokesGroup());
-                intersectedPolygonItem->strokesGroup()->addToGroup(polygonItem);
+                auto stuff = intersectedPolygonItem->strokesGroup();
```
Comment (multun) [inline src/domain/UBGraphicsScene.cpp:1056]: I added this by mistake, it's some unrelated diagnosis code

Comment (kaamui) [reply src/domain/UBGraphicsScene.cpp:1056]: :+1: 

Thank you

**Comment by multun** [comment]
It should be better now :) do you mind changing the default merge policy to rebase and merge? linear histories are generally regarded as easier to read

**Comment by kaamui** [comment]
I never though I could do a re-rebase after validation to make the commit message better, thanks for the advice, I'll do it on this one.

**Comment by Nikhil1404** [comment]
Can you make pip video camera widget application

**Comment by Nikhil1404** [comment]
Make something that anyone can add any application easily

**Comment by multun** [comment]
@Nikhil1404 can you create your own issue? It doesn't seem related to what's there
@kaamui would you mind merging this?

**Comment by Nikhil1404** [comment]
Can you make application for pip camera
